website/docs/guides/text-search.mdx

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 4
```

Comment:
Why 4 and not 3?
And while you are on it: please add `description:` to this page and `vector-search.mdx`, like we have in `replication.md`.